### PR TITLE
Feature/next auth

### PR DESCRIPTION
**Pull request by AGeck113**

### CODE REVIEW

File: components/Footer/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      ) : (
+        <section>
+          <p>No user logged in</p>
+        </section>
+      )}
+
+      {session ? (
+        <StyledImage
+          src={session.user.image}
+          alt="Profile Picture of the user"
```
Comment (doemser) [inline components/Footer/index.js:51]: Saying in an alt tag of an img that it is a picture is redundant. User name should be enough.

File: components/Login/index.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  font-size: 1.1rem;
+  font-weight: bold;
+`;
+export default function Login() {
+  const router = useRouter();
+
+  function handleSignIn() {
+    router.push("/");
+    signIn();
+  }
```
Comment (doemser) [inline components/Login/index.js:58]: you should be redirected to your "/" path automatically after signin in.

File: pages/api/userCars/[id].js [hunk 3]
```diff
@@ -1,28 +1,27 @@
...
+//           $set: request.body,
+//         }
+//       );
+//       if (!updatedCar) {
+//         return response.status(404).json({ status: "Not Found" });
+//       }
+//       return response.status(200).json(updatedCar);
+//     }
+//   }
+//   return response.status(403).json({ status: "Not logged in!" });
```
Comment (doemser) [inline pages/api/userCars/[id].js:26]: no commented code in production

**Review by doemser — CHANGES_REQUESTED** [review]
Good one, just found some little stuff. 

You are now predestined to help other students in the course.

File: components/Footer/index.js (hunk 1 above)
Comment (AGeck113) [reply components/Footer/index.js:51]: Thank you, done

File: components/Login/index.js (hunk 2 above)
Comment (AGeck113) [reply components/Login/index.js:58]: Done

File: pages/api/userCars/[id].js (hunk 3 above)
Comment (AGeck113) [reply pages/api/userCars/[id].js:26]: Forgot to delete, done

**Review by doemser — APPROVED** [review]

**Pull request merged**